app/src/main/res/layout/fragment_login.xml

```diff
@@ -4,8 +4,57 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:padding="50dp"
-    tools:context=".presentation.auth.sign_in.LoginFragment">
+        android:paddingStart="50dp"
+        android:paddingEnd="50dp"
+        android:paddingBottom="50dp"
+        tools:context=".presentation.auth.sign_in.LoginFragment">
+
+    <com.google.android.material.appbar.AppBarLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@android:color/transparent"
+            android:paddingStart="0dp"
+            android:paddingTop="0dp"
+            android:paddingEnd="0dp"
+            android:paddingLeft="50dp"
+            android:paddingRight="50dp"
+            app:liftOnScroll="true"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintBottom_toTopOf="@+id/imageView2"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent">
+
+        <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_weight="0.4"
+                android:background="@color/white"
+                android:orientation="horizontal"
+                android:padding="5dp">
+
+            <Button
+                    android:id="@+id/backBtn"
+                    style="@style/Widget.App.Button.OutlinedButton.IconOnly"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="bottom|end"
+                    android:layout_marginEnd="14dp"
+                    android:background="@color/white"
+                    app:icon="@drawable/ic_keyboard_backspace"
+                    app:iconTint="@color/green_dark" />
+
+            <TextView
+                    android:id="@+id/textHome"
+                    android:layout_width="wrap_content"
+                    android:layout_height="match_parent"
+                    android:gravity="center"
+                    android:text="Home"
```

Comment:
In this case will better use string resource (check string.xml file). And instead "Home" better to use Screen Name like "Sign in" or "Sign in with e-mail"